Annotation/Index.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Annotation;
+
+use Doctrine\Common\Annotations\Annotation\Attributes;
+
+/**
+ * Annotation to mark a class as an Elasticsearch index.
+ *
+ * @Annotation
+ * @Target("CLASS")
+ */
+final class Index extends AbstractAnnotation
+{
+    /**
+     * Index alias name. By default the index name will be created with the timestamp appended to the alias.
+     */
+    public $alias;
+
+    /**
+     * Index alias name. By default the index name will be created with the timestamp appended to the alias.
+     */
+    public $hosts = [
```

Comment:
@saimaz Is this a good idea? Its not possible to use environment variables or parameters inside annotation or?

Reply:
There will be a config which can override index settings. The annotation settings were added to make things more simple. In that case, you won't need any config someone uses default settings (like the host is 127.0.0.1:9200 and index alias name is like the class name).

Reply:
Ah ok, sounds good.

PS: the comment above $hosts looks false.

Reply:
This is why it is still a beta stage :) But if you could test this version and give some feedback I would appreciate it a lot.

Reply:
I will try to test the version at end of the week.

Reply:
At the end of week, there will be docs written as wellr I think.